functional/otlp/run_test_mode.sh

```diff
@@ -22,27 +22,33 @@ set -e
 
 [ -z "${SERVER_MODE}" ] && export SERVER_MODE="none"
 
+[ -z "${TEST_EXECUTABLE}" ] && echo "Please specify TEST_EXECUTABLE name" && exit 1
+
+[ -z "${TEST_URL}" ] && echo "Please specify TEST_URL endpoint (without scheme)" && exit 1
+
 export CERT_DIR=../cert
 
-export TEST_BIN_DIR=${BUILD_DIR}/functional/otlp/
+export TEST_BIN_DIR="${BUILD_DIR}/functional/otlp/"
+
+[ ! -f "${TEST_BIN_DIR}/${TEST_EXECUTABLE}" ] && echo "::notice::Executable ${TEST_EXECUTABLE} not built in this configuration" && exit 0
```

Comment:
Because this script gets its inputs from run_test.sh and all other CI checks that run these tests only build WITH_OTLP_HTTP, I thought this would be the easiest solution to supporting existing checks that only test HTTP functionality as well this check that does gRPC, as well.

For visibility, this notice is also shown in the annotations section:

![image](https://github.com/user-attachments/assets/4320d8e7-0c19-4109-b688-039653b9878b)